### PR TITLE
SRG-APP-000414-CTR-001010 is completely manual

### PR DESCRIPTION
**Pull request by jhrozek**
#### Description:

- This SRG is completely manual

#### Rationale:

- OCP4 STIG coverage

#### Review Hints:

- I will post the link to the SRG export later

### CODE REVIEW

File: controls/srg_ctr/SRG-APP-000414-CTR-001010.yml [hunk 1]
```diff
@@ -5,6 +5,25 @@ controls:
...
+
+    Bind to the appropriate cluster RBAC role
+
+    > oc adm policy add-cluster-role-to-user <ROLE> -z <NAME>
+
+    For more information see the following guides:
+
+    https://docs.openshift.com/container-platform/latest/authentication/using-rbac.html
+    https://docs.openshift.com/container-platform/latest/authentication/understanding-and-creating-service-accounts.html
+    https://docs.openshift.com/container-platform/latest/authentication/using-service-accounts-in-applications.html
```
Comment (Mab879) [inline controls/srg_ctr/SRG-APP-000414-CTR-001010.yml:29]: Please add a blank line at the end of the file.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Minor formatting issue. I have not reviewed the technical changes.

**Comment by jhrozek** [comment]
@rhmdnd @Vincent056 PTAL

**Review by Vincent056 — APPROVED** [review]
/lgtm

**Pull request merged**